src/glue/resources/dataset_crawler_assignments.yaml

```diff
@@ -0,0 +1,10 @@
+standard:
+  - answers_v1
+  - info_v2
```

Comment:
Why is this info_v2?

Reply:
Because v2 is the version in the [dataset_mapping.json](https://github.com/Sage-Bionetworks/BridgeDownstream/blob/main/src/glue/resources/dataset_mapping.json).